pom.xml

```diff
@@ -224,6 +224,7 @@
                            and there is very little chance this will get ever fixed. -->
                       <ignoreClass>org.w3c.dom.ElementTraversal</ignoreClass>
                       <ignoreClass>org.w3c.dom.UserDataHandler</ignoreClass>
+                      <ignoreClass>org.uberfire.java.nio.base.version.VersionRecord</ignoreClass>
```

Comment:
the ignoreClass should be explained and we usually do not do this for classes from this repository or from dependency we directly control. Please remove this